Source/Blazorise/Components/Accordion/Accordion.razor.cs

```diff
@@ -56,6 +58,35 @@ internal bool IsLastInAccordion( Collapse collapse )
         return false;
     }
 
+    internal void NotifyAccordionItemInitialized( AccordionItem accordionItem )
+    {
+        accordionItems ??= new();
+
+        accordionItems.Add( accordionItem );
+    }
+
+    internal void NotifyAccordionItemRemoved( AccordionItem accordionItem )
+    {
+        if ( accordionItems is not null && accordionItems.Contains( accordionItem ) )
+            accordionItems.Remove( accordionItem );
+    }
+
+    internal bool IsFirstInAccordion( AccordionItem accordionItem )
+    {
+        if ( accordionItems is not null && accordionItems.IndexOf( accordionItem ) == 0 )
+            return true;
+
+        return false;
+    }
+
+    internal bool IsLastInAccordion( AccordionItem accordionItem )
+    {
+        if ( accordionItems is not null && accordionItems.IndexOf( accordionItem ) == accordionItems.Count - 1 )
```

Comment:
I doubt it, but could this have a dumb bug like:
accordionItems being empty, 
which means  
**accordionItems.IndexOf( accordionItem )** => **-1**
**accordionItems.Count - 1** would also lead to **-1**

Reply:
If the accordionItems are empty, then it is already invalid. I would leave it like this. I don't want to introduce another bug by accident. If anyone complains in the future, we will change it.